src/Storage/Device/S3.php

```diff
@@ -659,9 +660,9 @@ private function getSignatureV4(string $method, string $uri, array $parameters =
      *
      * @return  object
      */
-    private function call(string $method, string $uri, string $data = '', array $parameters=[])
+    public function call(string $method, string $uri, string $data = '', array $parameters=[])
```

Comment:
```suggestion
    private function call(string $method, string $uri, string $data = '', array $parameters=[])
```
Please avoid changing the access  modifiers